spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -413,38 +392,6 @@ describe('document-capture/components/acuant-capture', () => {
       await waitForElementToBeRemoved(error);
     });
 
-    it('renders error message if capture succeeds but photo is too small', async () => {
-      const { getByText, findByText } = render(
-        <DeviceContext.Provider value={{ isMobile: true }}>
-          <AcuantContextProvider sdkSrc="about:blank">
-            <AcuantCapture label="Image" minimumFileSize={500 * 1024} />
-          </AcuantContextProvider>
-        </DeviceContext.Provider>,
-      );
-
-      initialize({
-        start: sinon.stub().callsFake(async (callbacks) => {
-          await Promise.resolve();
-          callbacks.onCaptured();
-          await Promise.resolve();
-          callbacks.onCropped({
-            glare: 70,
-            sharpness: 38,
-            image: {
-              data: 'data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg"/%3E',
-            },
-          });
-        }),
-      });
-
-      const button = getByText('doc_auth.buttons.take_picture');
-      fireEvent.click(button);
-
-      const error = await findByText('errors.doc_auth.photo_blurry');
```

Comment:
This is the correct test case to be removing here, but it wasn't testing the correct thing.

I confirmed in `master` with the following diff that the tests will pass as expected:

```diff
diff --git a/spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx b/spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx
index bf7b2ec3b..14ad9a4a3 100644
--- a/spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx
+++ b/spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx
@@ -429,7 +429,7 @@ describe('document-capture/components/acuant-capture', () => {
           await Promise.resolve();
           callbacks.onCropped({
             glare: 70,
-            sharpness: 38,
+            sharpness: 60,
             image: {
               data: 'data:image/svg+xml,%3Csvg xmlns="http://www.w3.org/2000/svg"/%3E',
             },
@@ -440,7 +440,7 @@ describe('document-capture/components/acuant-capture', () => {
       const button = getByText('doc_auth.buttons.take_picture');
       fireEvent.click(button);
 
-      const error = await findByText('errors.doc_auth.photo_blurry');
+      const error = await findByText('errors.doc_auth.photo_file_size');
 
       expect(error).to.be.ok();
     });
```

The test case to verify sharpness is still present after these changes:

https://github.com/18F/identity-idp/blob/0c2a67755059781fb5eb23b12fb77e891344ef79/spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx#L275

I assume it was a bad copy/paste on my part.